apps/emqx_dashboard/src/emqx_dashboard.erl

```diff
@@ -76,7 +76,8 @@ start_listeners(Listeners) ->
         security => [#{'basicAuth' => []}, #{'bearerAuth' => []}],
         swagger_global_spec => GlobalSpec,
         dispatch => dispatch(),
-        middlewares => [?EMQX_MIDDLE, cowboy_router, cowboy_handler]
+        middlewares => [?EMQX_MIDDLE, cowboy_router, cowboy_handler],
+        swagger_support => emqx:get_config([dashboard, swagger_support], true)
```

Comment:
Nit: It would be great if we could add a test case in the `emqx_dashboard_SUITE` to test update is valid when using emqx:update_config xxx